### PR TITLE
chore: add clarity script

### PR DESCRIPTION
**Pull request by afc163**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

https://clarity.microsoft.com/

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    --       |
| 🇨🇳 Chinese |    --       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .dumi/scripts/clarity.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// https://clarity.microsoft.com
+(function (c, l, a, r, i, t, y) {
+  c[a] =
+    c[a] ||
+    function () {
+      (c[a].q = c[a].q || []).push(arguments);
+    };
+  t = l.createElement(r);
+  t.async = 1;
+  t.src = 'https://www.clarity.ms/tag/' + i;
```
Comment (yoyo837) [inline .dumi/scripts/clarity.js:11]: 国内默认情况下访问不了这个东西

Comment (afc163) [reply .dumi/scripts/clarity.js:11]: 好像能访问：https://www.17ce.com/site/http/20240521_83c4c2f0171f11efb79925d58b52c95a:1.html

Comment (yoyo837) [reply .dumi/scripts/clarity.js:11]: 我24小时挂🪜，本打算推给公司用的，结果就是发现了不能，

切换redirect后：
image

Comment (afc163) [reply .dumi/scripts/clarity.js:11]: 没关系，GA 也不是 100% 稳定的。

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**